app/Resources/views/page/legislatives/candidat.html.twig

```diff
@@ -0,0 +1,79 @@
+{% extends 'page/legislatives/layout.html.twig' %}
+
+{% block page_title 'Mounir Mahjoubi | Législatives' %}
+{% block canonical_url url('site_legislatives_candidat') %}
+
+
+{% block content__inner %}
+
+    <div class="l__wrapper">
+        <a href="{{ path('site_legislatives_annuaire') }}" class="icon--with-text text--summary space--20-0">
+            {{ include('components/caret--left.html.twig') }}
+            Retour aux candidats du 78</a>
+    </div>
+
+    <div class="l__wrapper--medium space--30-0 l__row l__row--h-stretch l__mobile--col">
+        <div class="legislatives__candidate__photo progressive-background link--no-decor"
+             data-sd="{{ static_asset('elles-marchent/mounir.jpg', {'w': 100, 'q': 20}) }}"
+             data-hd="{{ static_asset('elles-marchent/mounir.jpg', {'w': 400, 'q': 70}) }}">
+        </div>
+        <div class="legislatives__candidate__info">
+            <h1 class="text--large">Mounir Mahjoubi</h1>
+            <span class="text--body">1e circonscription<span>
```

Comment:
Oh okay! I didn't know the values will come from the CMS. So then you don't need to worry about because people will not implement such details I think.